Applications/PlazaDetails/Strength.qmd

```diff
@@ -9,10 +9,10 @@ The following mentioned are among the popular services available in openEO algor
 The purpose of this document is to solely provide users with an assumption on how these credits work and their strength for varying services.
 
 ##### NDVI
-For the calculation of Normalized Difference Vegetation Index (NDVI), you can acquire ~250 NDVI images, each with an area of 1 hectare, i.e. 1 NDVI image patch of 1 hectare cost 3-5 credits.
+To calculate the Normalized Difference Vegetation Index (NDVI), approximately 250 NDVI images can be acquired, each covering an area of 1 hectare. Thus, one NDVI image patch of 1-hectare costs 3-5 credits.
 
 ##### NDII
-Computation of the Normalized Difference Infrared Index (NDII) provides the user with an indication of the water content in the plant canopies. A straightforward calculation of the NDII can be achieved from openEO algorithm plaza NDII service with the use of 4-8 credits per hectare area, i.e. you can use your 1000 credits for computing NDII in a given area of ~117 hectares.
+The computation of the Normalized Difference Infrared Index (NDII) offers insight into the water content in plant canopies. Using the openEO algorithm, plaza NDII service normally requires 4-8 credits per hectare area. Therefore, with 1000 credits, NDII computation can cover an area of approximately 117 hectares.
```

Comment:
see comment above, this is also not the place to introduce what NDII is used for